packages/teleterm/src/main.ts

```diff
@@ -31,13 +31,20 @@ const mainProcess = MainProcess.create({
   fileStorage,
 });
 
-// node-pty is not yet context aware
-app.allowRendererProcessReuse = false;
+const sharedProcess = fork(
+  path.join(__dirname, 'sharedProcess.js'),
+  [`--addr=${settings.shared.networkAddr}`],
+  {
+    stdio: 'inherit',
+  }
+);
+
```

Comment:
Should we move that inside `packages/teleterm/src/mainProcess/mainProcess.ts`? This is where the tshd process is started as well.

Reply:
Yes, done